docs/runtime-contract.md

```diff
@@ -282,9 +280,9 @@ code. These settings apply to both `livenessProbe` and `readinessProbe`:
 - `initialDelaySeconds` set to 0
 - `periodSeconds` set to platform-specific value
 
-In order to enable scaling in response to load, developers SHOULD NOT set the
-`initialDelaySeconds` to a value greater than 0, and should aim to minimize
-container startup time (aka cold start time).
+In order to enable scaling in response to load, setting `initialDelaySeconds`
+to a value greater than 0 can be used, while striving to minimize container
```

Comment:
I think this sentence now reads backwards -- it suggests that `initialDelaySeconds > 0` may improve scaling response, which is almost certainly not true.

Reply:
oops, yeah I got that backwards.... let me fix that